db/user.js

```diff
@@ -0,0 +1,113 @@
+// grab our db client connection to use with our adapters
+const client = require('./client');
+const bcrypt = require("bcrypt");
+const SALT_ROUNDS = 10;
+
+async function createUser({ username, password, email, isAdmin }) {
+  const hashPassword = await bcrypt.hash(password, SALT_ROUNDS);
+  try {
+    const {
+      rows: [user],
+    } = await client.query(
+      `
+        INSERT INTO users(username, password, email, "isAdmin") 
+        VALUES ($1, $2, $3, $4)
+        ON CONFLICT (username) DO NOTHING
+        RETURNING *;
+      `,
+      [username, hashPassword, email, isAdmin]
+    );
+
+    delete user.password;
+    return user;
+  } catch (error) {
+    throw error;
+  }
+}
+
+async function getAllUsers() {
+  try{
+  const { rows } = await client.query(
+      `SELECT *
+      FROM users;
+    `);
+  
+    return rows;
+  }catch (error){
+    throw error;
+  }
+}
+
```

Comment:
You're returning  all of the users passwords in this row query. I'd like see the user password deleted before returned